### ISSUE TITLE
pogo-feed.mmoex.com 404 issue

### ISSUE THREAD

**Issue by flo-security** [5andr0/PogoLocationFeeder#203]
I think it has something to do with the API. Can you check if it works after the 4/8 update? Or is it my pc....

**Comment by Demigodplayz** [comment]
Same here

**Comment by RaeneDev** [comment]
404..... Client-side Error which means that either the page has been removed or moved and the URL was not changed accordingly........

**Comment by halvors** [comment]
Really, "404 Fuck You"

**Comment by halvors** [comment]
Same problem here. Is this taken down for good, or? Is there an alternate service?

**Comment by flo-security** [comment]
Well. The new Necrobot works, Team UK6 cracked pogo again, www.pokesnipers.com works

**Comment by Avatar67** [comment]
How can I use www.pokesnipers.com instead of pogo-feed.mmoex.com?

**Comment by flo-security** [comment]
Im not the developer hahah ;P

**Comment by Rishabhpatted24** [comment]
@flo-security  any working free sniper bots mate?

**Comment by cannap** [comment]
This works for me https://github.com/5andr0/PogoLocationFeeder/releases/tag/v0.1.2

**Comment by flo-security** [comment]
Just tested it... Same 404 error. @Rishabhpatted24 yes you can use pokesniper2 fine. The API has been updated. They are working on it now

**Comment by flo-security** [comment]
https://twitter.com/pokesnipers

**Comment by Rishabhpatted24** [comment]
@flo-security  where can i download it?

**Comment by saahilz** [comment]
pogo feeder is not working

**Comment by cannap** [comment]
https://gyazo.com/92dc6be9c6063e6dab3f334c8840f39d v.0.1.2

**Comment by Meijerbink** [comment]
You can enable pokesnipers in the settings file, it is working for me

**Comment by saahilz** [comment]
i have no idea how is the feeder working for you dude , mine is not bringing any results i have used the link you provided @cannap

**Comment by cannap** [comment]
i dont know^^

**Comment by Meijerbink** [comment]
Change this to true:
"UsePokeSnipers": true, 

in the Config/config.json

**Comment by rherartt** [comment]
it worked for me too, but its slow

**Comment by saahilz** [comment]
my location feeder is connected but not getting any results and i have enabled pokesnipe in settings that is very weird

**Comment by cannap** [comment]
@saahilz use the soltution from @Meijerbink

**Comment by Rishabhpatted24** [comment]
@canap which solution? Poke feeder says waiting for location in mine..but no location recieved.

**Comment by saahilz** [comment]
i think i am just hopeless all the settings are correct but it is still a no go for me

**Comment by Rishabhpatted24** [comment]
@saahilz same here mate..;/

**Comment by flo-security** [comment]
Yup. Pokesnipers work. But it is very slow. Wait 5 minutes 
@Rishabhpatted24 If you want to, you can use pokesniper2 manually in combination with pokesnipers. It is still down but I shall provide link 4 download if its availible (need to update API)

**Comment by saahilz** [comment]
the old pokesniper2 work manually ?

**Comment by Rishabhpatted24** [comment]
@flo-security  i downloaded it...it got locations but it said no *the pokemon name hre * is here 
It got locattions of many ...i havent tried to manual method...seems too tedious.

**Comment by Rishabhpatted24** [comment]
@flo-security oh wait my sniper just started working...pokefeeder one..idk how but it did :) thanks anyway lol

**Comment by Rishabhpatted24** [comment]
@flo-security  it is extremely slow though...

**Comment by luumanhvn** [comment]
why my pogo feeder cant see pokemon on discord chat, last 1 hour? :(
http://2.pik.vn/2016f4176356-7d60-43fb-8c14-b7b20a599acb.jpg

**Comment by flo-security** [comment]
Run as Admin, check Win Firewall

**Comment by rahulrverma** [comment]
In log it says:
[DEBUG] Pokesnipers API error: The remote server returned an error: (403) Forbidden.

**Comment by djdookie** [comment]
Does it work for you @flo-security?

**Comment by TemhAAhmeT** [comment]
for me pokesnipe works but discord doesnt

**Comment by mhui1125** [comment]
@TemhAAhmeT same

**Comment by JimmyNeutrino** [comment]
Can u explain me how to connect pogo feeder with pokesnipe? My feed is empty right now

**Comment by djdookie** [comment]
read above:

> 
Change this to true:
"UsePokeSnipers": true,
in the Config/config.json

**Comment by FVKUALL** [comment]
works through setting:

Change this to true:
"UsePokeSnipers": true,
in the Config/config.json

**Comment by rherartt** [comment]
@FVKUALL the latest version?

**Comment by isolah123** [comment]
yes we can snipe it automatically, yes we can transfer it automatically. question is how many balls u have? because we need pokestop to swap right?

**Comment by Tavorib** [comment]
ETA for other servers to work

**Comment by ExcuseMi** [comment]
Working !

**Comment by Jketchup89** [comment]
hey anyone nothing see a charizard or Alakazam? if possible also Vileplume
